.github/workflows/generate_app_id.yml

```
@@ -19,7 +19,7 @@ jobs:
         steps:
             - name: Capture Vercel preview URL
               id: vercel_preview_url
-              uses: binary-com/vercel-preview-url-action@v1.0.5
+              uses: binary-com/vercel-preview-url-action@v1.0.6
```

```suggestion
              uses: binary-com/vercel-preview-url-action@master
```
Can we use the master branch so that we don't have to update the version everytime?